package.json

```diff
@@ -13,15 +13,17 @@
     "coverage": "IASQL_ENV=test jest -i --coverage --forceExit",
     "coverage:local": "IASQL_ENV=test export $(cat .env | xargs) && yarn coverage",
     "serve": "node --max_old_space_size=8192 dist/index.js",
-    "serve:debug": "node --inspect=0.0.0.0:9229 --max_old_space_size=8192 dist/index.js",
+    "serve:debug": "yarn build && node --max_old_space_size=8192 dist/index.js",
     "start": "yarn serve",
-    "start:debug": "nodemon -e ts -w ./src -x yarn serve:debug",
+    "start:debug": "nodemon --inspect=0.0.0.0:9229 -e ts -w ./src -x yarn serve:debug",
```

Comment:
We should `EXPOSE` port 9229, as well, so we can hook up the Chrome Node Debugger to the engine during dev.